hex/helpers/build

```diff
@@ -10,7 +10,14 @@ fi
 install_dir="$DEPENDABOT_NATIVE_HELPERS_PATH/hex"
 mkdir -p "$install_dir"
 
-mix local.hex --force
+# Initial Hex install
+mix local.hex --force --if-missing
+# Annoyingly, the Hex version cannot be specified during the initial install.
+# It must be downgraded later.
+if [ -n "$HEX_VERSION" ]; then
+  mix hex.install "$HEX_VERSION"
```

Comment:
This will fail if `mix local.hex` isn't called beforehand... and that command updates to the latest Hex... so if it's broken out of the gate, we will be hosed. 😢 

That's why we want the pinned install to happen immediately after the `mix local.hex` install.